hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/workload/selective/BalancingJob.java

```diff
@@ -51,12 +51,12 @@ public void run() {
         try {
             if (consumersRegistry.isLeader()) {
                 try (Timer.Context ctx = metrics.consumersWorkloadRebalanceDurationTimer(kafkaCluster).time()) {
-                    logger.info("Initializing workload balance.");
+//                    logger.info("Initializing workload balance.");
```

Comment:
Hey, I wonder why you commented these. I noticed they disappear after being filtered out by logback. We should think whether they're needed and if so, not filter them out. I feel this is useful info to find out when the balancing happened.